docs/tutorials/tutorial_params_options_simulate.ipynb

```diff
@@ -0,0 +1,973 @@
+{
```

Comment:
<p>I think the first paragraph should be in <code>tutorials/index.rst</code>.  Could you merge this and the existing comment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/respy/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>